quiz/service.py

```diff
@@ -0,0 +1,139 @@
+import json
+
+from django.contrib.auth import get_user_model
+from django.db.models import Q
+from django.template.defaultfilters import register
+
+from feedbacks.models import Feedback
+
+__all__ = ['check_group_indicators', 'get_average_results',
+           'get_indicators_values', 'get_groups_results',
+           'get_final_buisness_result',
+           'get_feedback', 'zip_list', 'NUMBER_OF_QUESTION_FOR_BUISNESS_QUIZ']
+
+NUMBER_OF_QUESTION_FOR_BUISNESS_QUIZ = 24
+
+
+def check_group_indicators(group_indicator):
+    """Return dictionary with correct indicators
+    by adding constant
+    if indicator is < 0 then shift value of indicator to 0
+    if is indicator is > 100 then shift value of indicator to 100
+    :param dict group_indicator: Dictionary with group indicators
+    :return:dict Dictionary with correct indicators
+    """
+    for indicator in group_indicator:
+        if group_indicator[indicator] < 0:
```

Comment:
_From 26 to 29 line_
I think it would be better to transfer these magic numbers to a constant, for example, MAX_INDICATOR_VALUE, MIN_INDICATOR_VALUE. Like a NUMBER_OF_QUESTION_FOR_BUISNESS_QUIZ.